recipe/meta.yaml

```diff
@@ -108,16 +108,16 @@ outputs:
         # shared
         - test -f $PREFIX/lib/libfaiss.so                        # [linux]
         - test -f $PREFIX/lib/libfaiss.dylib                     # [osx]
-        - if not exist %PREFIX%\\Library\\bin\\faiss.dll exit 1  # [win]
+        - if not exist %PREFIX%\Library\bin\faiss.dll exit 1     # [win]
 
         # absence of static libraries
         - test ! -f $PREFIX/lib/libfaiss.a                       # [not win]
-        - if exist %PREFIX%\\Library\\lib\\faiss.lib exit 1      # [win]
+        - if exist %PREFIX%\Library\lib\faiss.lib exit 1         # [win]
 
         # headers
         {% for each_header in headers %}
         - test -f $PREFIX/include/faiss/{{ each_header }} || (echo "{{ each_header }} not found" && exit 1)  # [unix]
-        - if not exist %LIBRARY_INC%\\faiss\\{{ "\\".join(each_header.split("/")) }} exit 1                  # [win]
+        - if not exist %LIBRARY_INC%\faiss\{{ r"\".join(each_header.split("/")) }} exit 1                    # [win]
```

Comment:
I'm not sure if Jinja supports raw strings.

Reply:
nope, apparently, it doesn't

sorry for the noise

Reply:
All good, let's call it "experimental science" ;).